src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1306,6 +1306,13 @@
           ]
         },
 
+        "rank": {
+          "description": "A number representing the priority or importance of the result.",
```

Comment:
>or impo [](start = 61, length = 7)

why do you need this? are you comfortable deleting 'or importance'? importance is a slippery slope that pulls in the severity of vulns, the level of exposure, etc. rank is a relative ordering. i think 'priority' sufficiently communicates what we're after.